webserver/pkgpkr/webservice/templates/webservice/index.html

```diff
@@ -43,9 +43,10 @@ <h2 id="home-title">Try PkgPkr!</h2>
                                     <button id="recommendation-button" class="button is-large is-link">Get Recommendation</button>
                                 </div>
                                 <div id="recommendation-hyperlink-div">
-                                    <a id="recommendation-hyperlink-a" target="_blank" href="{% url 'about' %}">
-                                        How do we generate our recommendations?
+                                    <b><a id="recommendation-hyperlink-a" target="_blank" href="{% url 'about' %}">
```

Comment:
![Capture](https://user-images.githubusercontent.com/26328949/80915619-826f7080-8d21-11ea-9b46-2af58845fc2f.PNG)

For some reason, this ver breaks the layout of index.html, but I will figure out and fix it